pkg/provider/kv/kv.go

```diff
@@ -143,14 +144,27 @@ func (p *Provider) watchKv(ctx context.Context, configurationChan chan<- dynamic
 	return nil
 }
 
+func (p *Provider) configKeyPrefix() string {
+	aliasKey := p.RootKey + "/alias"
+
+	res, err := p.kvClient.Get(aliasKey, nil)
+	if err != nil {
```

Comment:
Instead of always ignoring the error, we could only ignore errors that indicate that the entry does not exist (if the kvClient has an error type for that), all others could be returned from buildConfiguration().